github_issue_summarization/02_training_the_model_tfjob.md

```diff
@@ -74,12 +77,12 @@ kubectl --namespace=${NAMESPACE} create secret generic user-gcp-sa --from-file=u
 
 ### Run the TFJob using your image
 
-[ks-kubeflow](ks-kubeflow) contains a ksonnet app to deploy the TFJob.
+[ks_app](ks_app) contains a ksonnet app to deploy the TFJob.
 
-Set the appropriate params for the tfjob component
+Set the appropriate params for the tfjob component:
 
-```commandline
-cd ks-kubeflow
+```bash
+cd ks_app
 ks param set tfjob namespace ${NAMESPACE} --env=${KF_ENV}
```

Comment:
I think namespace is set based on the environment and is no longer an explicit parameter.
https://github.com/kubeflow/examples/blob/79904082079b56a7c0fd8a009855d8fa6be97bb9/github_issue_summarization/ks-kubeflow/components/tfjob-v1alpha2.jsonnet#L7

So we should probably delete the line 
ks param set tfjob namespace ...

If users want to set the namespace they should do this by setting the namespace on their environment (they may need to create a new environment for that).